client/src/elm/Pages/MessagingCenter/View.elm

```diff
@@ -506,18 +504,14 @@ viewResilienceMessage language nurseId nurse model ( messageId, message ) =
                             head
                         ]
             in
-            if viewOptions then
-                div [ class "title-wrapper" ]
-                    [ plainTitle
-                    , div
-                        [ class "options"
-                        , onClick <| SetMessageOptionsDialogState <| Just <| MessageOptionsStateMain ( messageId, message )
-                        ]
-                        [ text "OP" ]
+            div [ class "title-wrapper" ]
+                [ plainTitle
+                , div
+                    [ class "options"
+                    , onClick <| SetMessageOptionsDialogState <| Just <| MessageOptionsStateMain ( messageId, message )
                     ]
-
-            else
-                plainTitle
+                    [ text "OP" ]
```

Comment:
@anvmn Do you plan to translate this text in the future?

Reply:
@mariano-dagostino This will be replaced by icon